packages/docs/docs/crawler/starting.md

```diff
@@ -17,9 +17,9 @@ findkit crawl start
 
 This will re-index every page on the site every time.
 
-### Partial
+### Updated
 
-Crawl only the pages that have updated content:
+Start crawl on pages that have updated content:
```

Comment:
Alkuperäinen on kyllä tarkempi. Crawleri oikeasti crawlaa vaan ne sivut. Tää muutos viittaa siihen että se voi crawlata muutakin.

Reply:
Other methods of crawling were documented to start with "Start crawl" to match CLI command.

Maybe `Start crawl on updated pages only`? @esamattis